python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -278,12 +307,16 @@ def _setupDatabase(configurable):
         A ConfigurableInstance with a database-managing class in its ``target``
         field. The API of ``target`` must expose a ``create_tables`` method
         taking no arguments.
```

Comment:
This documentation is out of date. See my proposed change to `ApPipeConfig.ppdb` before updating, though.

Reply:
Now that the `config.apply` method I've removed this part of the code.